src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -229,6 +231,38 @@ public DocumentConventions()
             }
         }
 
+        /// <summary>
+        /// Set the timeout for the second broadcast attempt.
+        /// Default: 30 Seconds.
+        ///
+        /// Upon failure of the first attempt the request executor will resend the command to all nodes simultaneously. 
+        /// </summary>
+        public TimeSpan? SecondBroadcastAttemptTimeout
+        {
+            get => _secondBroadcastAttemptTimeout;
+            set
+            {
+                AssertNotFrozen();
+                _secondBroadcastAttemptTimeout = value;
+            }
+        }
+
+        /// <summary>
+        /// Set the timeout for the first broadcast attempt.
+        /// Default: 5 Seconds.
+        ///
+        /// First attempt will send a single request to a selected node.
+        /// </summary>
+        public TimeSpan? FirstBroadcastAttemptTimeout
```

Comment:
Set defaults in `DocumentConventions` ctor and remove nullable

Reply:
deon